readthedocs/projects/tasks/builds.py

```diff
@@ -626,6 +630,23 @@ def get_valid_artifact_types(self):
                         },
                     )
 
+                # Rename file as "<project_slug>-<version_slug>.<artifact_type>",
+                # which is the filename that Proxito serves for offline formats.
+                filename = os.path.join(artifact_directory, list_dir[0])
+                _, extension = filename.rsplit(".", maxsplit=1)
+                output = os.path.join(
+                    self.data.project.checkout_path(self.data.version.slug),
+                    "_readthedocs/",
+                )
+                assert_path_is_inside_expected_path(Path(filename), Path(output))
```

Comment:
This check can be bypassed if `_readthedocs` itself is a symlink. We don't really care if the directory/file is outside `_readthedocs`, we only care that it isn't outside the docroot.

Reply:
I tried calling `assert_path_is_inside_docroot` before writing this extra function, but it didn't work. It raises the exception saying the path is not inside the DOCROOT 🤷🏼 . I'm not sure what to here. Would you mind jumping into this?

Reply:
Oh, I found the issue with the initial approach. It seems I was using a relative path and that made the function to get confused somehow. I'm going to push the change now.